wpimath/src/main/java/edu/wpi/first/math/filter/SlewRateLimiter.java

```diff
@@ -82,4 +82,28 @@ public void reset(double value) {
     m_prevVal = value;
     m_prevTime = MathSharedStore.getTimestamp();
   }
+
+  /**
+   * Sets the rate-of-change limit.
+   *
+   * @param positiveRateLimit The rate-of-change limit in the positive direction, in units per
+   *     second. This is expected to be positive.
+   * @param negativeRateLimit The rate-of-change limit in the negative direction, in units per
+   *     second. This is expected to be negative.
+   */
+  public void setLimit(double positiveRateLimit, double negativeRateLimit) {
+    m_positiveRateLimit = positiveRateLimit;
+    m_negativeRateLimit = negativeRateLimit;
+  }
+
+  /**
+   * Sets the rate-of-change limit in both directions.
+   *
+   * @param rateLimit The rate-of-change limit in both directions, in units per second. This is
+   *     expected to be positive.
+   */
+  public void setLimit(double rateLimit) {
+    m_positiveRateLimit = rateLimit;
+    m_negativeRateLimit = rateLimit;
```

Comment:
```suggestion
    m_negativeRateLimit = -rateLimit;
```
You could also make the first sentence of the doc comment match the first sentence of the `SlewRateLimit(double)` doc comment, but that's up to you.